src/jats/jats-exporter.ts

```diff
@@ -2326,4 +2322,13 @@ export class JATSExporter {
       fn.appendChild(this.document.createElement('p'))
     )
   }
+
```

Comment:
Sometimes we receive a table-wrap-foot that has an empty paragraph, and that paragraph is deleted based on the [paragraph rules](https://github.com/Atypon-OpenSource/manuscripts-transform/blob/4c085161d6aa7d349adba5750beb4dc9816d26ad/src/jats/jats-exporter.ts#L1129). I'm not sure if we need to keep it that way or if it was needed for a specific reason.